src/index.js

```diff
@@ -268,17 +268,20 @@ class MiniCssExtractPlugin {
         const { hashFunction, hashDigest, hashDigestLength } = outputOptions;
         const hash = createHash(hashFunction);
 
-        for (const m of this.getChunkModules(chunk, chunkGraph)) {
-          if (m.type === MODULE_TYPE) {
+        const modules = Array.from(
+          this.getChunkModules(chunk, chunkGraph)
+        ).filter((module) => module.type === MODULE_TYPE);
+        if (modules.length) {
```

Comment:
Let's use `continue`, it is faster, just put `// eslint-ignore` if linter will report about it (we need disable this rule, because we have more problems with it)